storybook/src/components/FieldSet/FieldSet.docs.mdx

```diff
@@ -44,6 +51,13 @@ Even so, always set `aria-required` on the individual Radios as well to ensure t
 
 <Canvas of={FieldSetStories.RadioGroup} />
 
+### Radio group with Hint
+
+A Field Set with a Radio group can also indicate whether all of the radios it contains are <em>not</em> required.
+Make sure to add the `required` or `aria-required="true"` attribute to <em>any</em> samely named Radios in this Field Set.
+
+<Canvas of={FieldSetStories.RadioGroupWithHint} />
+
```

Comment:
Is this story necessary? It isn't really any different from the 'With Hint' story, right?

Reply:
Removed